src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/in/Microsoft.Android.Sdk.BundledVersions.in.targets

```diff
@@ -17,6 +17,18 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <_AndroidRuntimePackId Condition=" '$(_AndroidRuntimePackId)' == '' ">$(_AndroidTargetingPackId)</_AndroidRuntimePackId>
   </PropertyGroup>
   <ItemGroup>
+    <!-- TODO: NET7TODO: Is this the right way to support building net6.0 projects? Need to bump versions as packages are release to NuGet.org as well -->
+    <KnownFrameworkReference
+        Include="Microsoft.Android"
+        TargetFramework="net6.0"
+        RuntimeFrameworkName="Microsoft.Android"
+        LatestRuntimeFrameworkVersion="32.0.300-rc.1.4"
+        TargetingPackName="Microsoft.Android.Ref.$(_AndroidTargetingPackId)"
+        TargetingPackVersion="32.0.300-rc.1.4"
+        RuntimePackNamePatterns="Microsoft.Android.Runtime.$(_AndroidRuntimePackId).**RID**"
+        RuntimePackRuntimeIdentifiers="android-arm;android-arm64;android-x86;android-x64"
+        Profile="Android"
+    />
```

Comment:
I wasn't sure if this was the right thing to do for supporting `net6.0` project builds with what will now be a `net7.0` SDK. Should I file something in dotnet/sdk for discussing this?

Reply:
Let me review this after merging... I suspect it is more complicated than just this. I think we need to somehow import the .NET 6 Microsoft.Android.Sdk pack to use our .NET 6 MSBuild tasks.